### PR TITLE
AlertDialog ui fixes

### LINKED ISSUE #110: Logout Dialog is not rounded like other dialogs in the app.
**Describe the bug**
AlertDialog Typically used in conjunction with [showDialog].

**To Reproduce**
Open App>FloatingActionButton>DiaLog

**Expected behavior**
AlertDialog should have a shape and contentPadding.
**Demonstration**
Before            |  After
:-------------------------:|:-------------------------:
27e0e8cb-80a8-4671-a372-a10ffbeb5f26 | 05eab1c0-1d37-468b-8ebd-228aa5b70e0c

**Environment**
- Device/emulator being used: <device version/emulator>
- OS (iOS/ Android); If Android then android version: <OS: OS version>
- Flutter version: <2.8>

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by Pavel401**
Fixes #110 

Before            |  After
:-------------------------:|:-------------------------:
27e0e8cb-80a8-4671-a372-a10ffbeb5f26 | 05eab1c0-1d37-468b-8ebd-228aa5b70e0c

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,43 +9,47 @@ environment:
   sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
-  connectivity_plus: ^1.0.2
-  cupertino_icons: ^1.0.2
-  date_time_picker: ^2.0.0
-  duration_picker: ^1.1.0+1
   flutter:
     sdk: flutter
```
Comment (ItsAdityaKSingh) [inline pubspec.yaml:13]: Here the sorted dependencies have been randomized again. I think keeping them sorted would be better as present in the codebase.

File: pubspec.yaml [hunk 2]
```diff
@@ -9,43 +9,47 @@ environment:
...
   shared_preferences: ^2.0.6
-  sizer: ^2.0.15
-  skeleton_text: ^1.0.0
-  sliding_up_panel: ^2.0.0+1
+  graphql_flutter: ^5.0.0
+  get_it: ^7.1.3
+  provider: ^5.0.0
   uni_links: ^0.5.1
-
+  share_plus: ^3.0.4
```
Comment (ItsAdityaKSingh) [inline pubspec.yaml:30]: Same for all these.

**Review by ItsAdityaKSingh — COMMENTED** [review]
The dependencies could be left sorted

File: pubspec.yaml (hunk 1 above)
Comment (Pavel401) [reply pubspec.yaml]: I have Sorted dependencies in the latest commit.

**Review by nb9960 — COMMENTED** [review]
This PR supersedes the changes by PR #87.

File: pubspec.yaml [hunk 3]
```diff
@@ -20,7 +20,10 @@ dependencies:
   flutter_local_notifications: ^9.2.0
   flutter_polyline_points: ^1.0.0
   fluttertoast: ^8.0.3
-  geocoder: ^0.2.1
+  geocoder:
```
Comment (ItsAdityaKSingh) [inline pubspec.yaml:23]: These changes are already present in #87. You may want to keep this PR for the issue itself :)

File: pubspec.lock [hunk 4]
```diff
@@ -362,10 +362,12 @@ packages:
   geocoder:
     dependency: "direct main"
     description:
-      name: geocoder
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "0.2.1"
+      path: "."
```
Comment (ItsAdityaKSingh) [inline pubspec.lock:365]: These changes too.

**Comment by nb9960** [comment]
You removed gradle files and there some conflicts, make sure to fix them

**Comment by ItsAdityaKSingh** [comment]
@Pavel401 Could you please fix the errors for the changes to be reviewed and merged?

**Comment by ItsAdityaKSingh** [comment]
@Pavel401 Could you please fix the build issues and conflicts for proper reviewing?

**Comment by ItsAdityaKSingh** [comment]
#161 Already resolves the changes proposed in this. @nb9960 Ig this can be closed :)

**Comment by ItsAdityaKSingh** [comment]
Closing as already resolved in #161!

**Pull request closed without merging**